src/index.html

```diff
@@ -1,13 +1,287 @@
+<!--#region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Bike</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="/images/logo/logo.svg">
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!--#endregion -->
+  <body class="page__body">
+
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              href="#"
+              class="top-bar__logo">
```

Comment:
If there are only 1 or 2 attributes, then there is no need to move them to new lines, you need to fix it everywhere
```suggestion
            <a href="#" class="top-bar__logo">
```